recipes/openh264/all/conanfile.py

```diff
@@ -6,120 +6,163 @@
-                        args.append('TARGET=%s' % target)
-                        tools.replace_in_file(os.path.join("codec", "build", "android", "dec", "jni", "Application.mk"),
-                            "APP_STL := stlport_shared",
-                            "APP_STL := %s" % str(self.settings.compiler.libcxx))
-                        tools.replace_in_file(os.path.join("codec", "build", "android", "dec", "jni", "Application.mk"),
-                            "APP_PLATFORM := android-12",
-                            "APP_PLATFORM := %s" % target)
-                        args.append("CCASFLAGS=$(CFLAGS) -fno-integrated-as")
-                env_build.make(args=args, target="libraries")
-                args.append('install-' + ('shared' if self.options.shared else 'static-lib'))
-                env_build.make(args=args)
+                env_build.make(args=self._make_args, target=self._library_filename)
 
     def package(self):
         self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
-        if self.options.shared:
-            exts = ['*.a']
-        else:
-            exts = ['*.dll', '*.so*', '*.dylib*']
-        for root, _, filenames in os.walk(self.package_folder):
-            for ext in exts:
-                for filename in fnmatch.filter(filenames, ext):
-                    os.unlink(os.path.join(root, filename))
-        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
+            with tools.chdir(self._source_subfolder):
+                env_build = AutoToolsBuildEnvironment(self)
+                env_build.make(args=self._make_args, target="install-" + ("shared" if self.options.shared else "static-lib"))
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        if self.settings.compiler == 'Visual Studio' and self.options.shared:
-            self.cpp_info.libs = ['openh264_dll']
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            self.cpp_info.libs = ["openh264_dll"]
         else:
-            self.cpp_info.libs = ['openh264']
+            self.cpp_info.libs = ["openh264"]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs.extend(['m', 'pthread'])
+            self.cpp_info.system_libs.extend(["m", "pthread"])
         if self.settings.os == "Android":
             self.cpp_info.system_libs.append("m")
-        libcxx = self.settings.get_safe("compiler.libcxx")
-        if libcxx in ["libstdc++", "libstdc++11"]:
-            self.cpp_info.system_libs.append("stdc++")
-        elif libcxx == "libc++":
-            self.cpp_info.system_libs.append("c++")
-        elif libcxx in ["c++_static", "c++_shared"]:
-            self.cpp_info.system_libs.extend([libcxx, "c++abi"])
+        self.cpp_info.names["pkg_config"] = "libopenh264"
```

Comment:
@madebr Why this change ? it does not seem conform to upstream https://github.com/cisco/openh264/blob/master/openh264.pc.in